File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(numList) {
    const answer = [0, 0];
    let evenCount = 0;
    let oddCount = 0;
    // 홀수와 짝수를 세어줄 변수 선언
    
    for (let i = 0; i < numList.length; i++) {
        if (numList[i] % 2 === 0) {
            evenCount++;
            // 2로 나눈 나머지가 0이면 짝수이므로 even이 카운트되고
        } else {
            oddCount++;
            // 나머지가 0이 아니라면 홀수이므로 odd가 카운트 된다
        }
    }

    answer[0] = evenCount;
    answer[1] = oddCount;
    // even, odd을 answer에 넣어준다
    return answer;
}

File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[2];
        int holl = 0;
        int zzak = 0;
        // 홀수와 짝수를 세어줄 변수 선언
        for(int i = 0; i < num_list.length; i++) {
            if ( num_list[i] % 2 == 0 ) {
                holl++;
            }   // 2로 나눈 나머지가 0이면 짝수이므로 holl이 카운트되고
            else {
                zzak++;
            }   // 나머지가 0이 아니라면 홀수이므로 zzak이 카운트 된다
        }
        answer[0] = holl;
        answer[1] = zzak;
        // holl, zzak을 answer에 넣어준다
        return answer;
    }
}